g11n-ws/tools/tool-cldr-extractor/src/main/java/com/vmware/i18n/utils/CLDRUtils.java

```diff
@@ -563,6 +572,43 @@ public static String readZip(String fileName, String zipPath) {
         return erasMap;
     }
 
+    /**
+     * Get the first day of one Country, the implementation is according to https://www.unicode.org/reports/tr35/tr35-49/tr35-dates.html#Week_Data
+     *
+     * @param localeStr
+     * @param firstDayData
+     * @return
+     */
+    private static Integer firstDayExtract(String localeStr, Map<String, String> firstDayData){
```

Comment:
recommend the method name firstDayExtract to getFirstDayExtract

Reply:
Just for keep code style consistent with previous code, but it really doesn't conform the naming convention, so rename it to 'extractFirstDay'.